ConsoleFollower/Program.cs

```diff
@@ -26,8 +40,20 @@ public class Program {
     // or go to https://www.sec.gov/edgar/searchedgar/companysearch.html and put the name of the investor you are interested in
     public static void Main(string[] args)
     {
-        if (args.Count() != 1) throw new Exception("Usage: Follower Cik");
-        var result =  GuruLoader.FetchDisplayPortfolio(args[0]).Result;
-        Console.WriteLine(DisplayPortToString(result));
+        var banner = "Usage: Follower Cik [-All]";
+        if (args.Count() > 2) throw new Exception(banner);
+        if (args.Count() == 2 && args[1] != "-All") throw new Exception(banner);
```

Comment:
Generally you don't want to throw a base 'Exception' type. It would be better to throw something more specific like ArgumentException.